core/themes/seven/css/style.css

```diff
@@ -777,8 +777,15 @@ div.teaser-checkbox .form-item,
 .text-format-wrapper .form-item {
   padding-bottom: 0;
 }
+.form-item {
+  margin-top: .5em;
+  margin-bottom: .5em;
+}
 .form-item label {
-  margin: 0 0 10px;
+  margin: 0 0 5px;
+}
```

Comment:
These styling changes seem like they would affect every form in the admin area. Are these necessary?

Reply:
Necessary no, no CSS changes are ever necessary. But this does fix form-element styling across all of Seven, which was a little out of whack. I'm happy to do that in a separate issue, but the only place in core that we can see it's so funky is on this page.